UseLatexMk.cmake

```diff
@@ -217,10 +213,21 @@ function(add_latex_document)
     set(ENV_COMMAND ${CMAKE_COMMAND} -E env openout_any="a")
   endif()
 
-  # Get an absolute path to the source
-  set(LMK_SOURCE_REPLACED ${CMAKE_CURRENT_BINARY_DIR}/${LMK_TARGET}_source.cc)
+  # For LaTeX package minted, replace value of outputdir from CMAKE_CURRENT_BINARY_DIR to
+  # its actual value by coping the LaTeX file to the build dir
+  set(LMK_SOURCE_REPLACED ${CMAKE_CURRENT_BINARY_DIR}/${LMK_SOURCE})
+  # for in-source-build, adjust
+  if("${CMAKE_CURRENT_BINARY_DIR}" STREQUAL "${CMAKE_CURRENT_SOURCE_DIR}")
+    get_filename_component(source_filename ${LMK_SOURCE} NAME_WE)
+    get_filename_component(source_extension ${LMK_SOURCE} LAST_EXT)
+    set(LMK_SOURCE_REPLACED "${CMAKE_CURRENT_BINARY_DIR}/${source_filename}_source${source_extension}")
+  endif()
```

Comment:
@dokempf We could search the LaTeX source file and check whether it conains "@CMAKE_CURRNET_SOURCE_DIR@". Only in this case, we have to do the `configure_file` and renaming steps.
This could help with in-source builds to get the same file name, unless they really use minted. I would prefer this but I wanted to know your opinion.